lib/routes/commitments.js

```diff
@@ -2,10 +2,21 @@ const router = require('koa-router')()
 const models = require('../models')
 
 router
-  .get('/commitments/participant/:participant', async (ctx) => {
-    ctx.body = await models.db.participant_event.findAll({
-      where: {
-        participant_id: ctx.params.participant
+  .get('/commitments/participant/:fbid', async (ctx) => {
+    let fbid = ctx.params.fbid
+    await models.db.participant.findOne({
+      where: {fbid}
+    }).then(async (participant) => {
+      if (participant) {
+        ctx.body = await models.db.participant_event.findAll({
+          where: {participant_id: participant.id}
+        })
+      } else {
+        ctx.status = 204
```

Comment:
I think this should actually return 400 to match the other "does not exist" errors

Reply:
That sounds reasonable.  Changed.